neqo-common/src/udp.rs

```diff
@@ -31,6 +32,7 @@ impl Socket {
         Ok(Self {
             state: quinn_udp::UdpSocketState::new((&socket).into())?,
             socket: tokio::net::UdpSocket::from_std(socket)?,
+            recv_buf: vec![0; u16::MAX as usize],
```

Comment:
This is too large.  I get that we don't want to overflow, but unless we are receiving multiple datagrams, we only need 2k or so (a constant might be a good idea for that limit).

Reply:
Why is this too large? This is about ~43 packets, which is smaller than realistic WAN congestion windows. Are you worried about Fx memory usage of this is per-socket?

Reply:
Ignore the comment.  I was reading this patch on its own and missed the follow-up that includes multiple packets.

I do think that a constant is the right thing to use for this, rather than the (rather arbitrary) `u16::MAX`.

Reply:
> I do think that a constant is the right thing to use for this, rather than the (rather arbitrary) `u16::MAX`.

:+1: I prepared https://github.com/mozilla/neqo/pull/1709.

> I was reading this patch on its own and missed the follow-up that includes multiple packets.

For others to follow, this is  https://github.com/mozilla/neqo/pull/1708.